line_profiler/line_profiler.py

```diff
@@ -6,13 +6,13 @@
 import tempfile
 import os
 import sys
-from io import StringIO
 from argparse import ArgumentError, ArgumentParser
 
-from IPython.core.magic import (Magics, magics_class, line_magic)
-from IPython.core.page import page
-from IPython.utils.ipstruct import Struct
-from IPython.core.error import UsageError
+try:
+    from ipython_extension import load_ipython_extension
```

Comment:
Should, at the very least, be `from .ipython_extension import load_ipython_extension`

However, this is a circular import, and while that can be ok, I'd rather avoid it for code readability. If I understand correctly, the important thing is that there is a `load_ipython_extension` function exposed at the top level.

Reply:
Indeed, I forgot the leading dot, thanks for noticing.
I removed the circular import by not importing ``load_ipython_extension`` in ``line_profiler.py`` at all.